### DISCUSSION TITLE
Does Chakra work with MS Edge <=18?

### DISCUSSION THREAD

**Discussion by doender** [chakra-ui/chakra-ui#4316] [Q&A]
We're seeing problems with our app on legacy MS Edge 18 ('pre-chrome') browsers. Most CSS variables don't seem to be loaded, for example `--chakra-space-1`. On the other hand, some *are* loaded, such as `--chakra-fontWeights-medium`.

Is this behavior known?
![image](https://user-images.githubusercontent.com/4175777/124257279-8321b380-db2c-11eb-8dec-a75849fadd66.png)

**Comment by with-heart** [comment]
@doender imo this is worth opening as an issue instead of a discussion. Saw your screenshot of the Chakra docs on Discord and that's quite a mess. I'm not entirely sure what's happening here though so we'd need to investigate.
